sql/plsql/PlSqlLexer.g4

```diff
@@ -761,6 +761,7 @@ INTERVAL                       : 'INTERVAL';
 INT                            : 'INT';
 INTO                           : 'INTO';
 INVALIDATE                     : 'INVALIDATE';
+INVALIDATION                   : 'INVALIDATION';
```

Comment:
Please add the new token to [non_reserved_keywords_in_12c](https://github.com/antlr/grammars-v4/blob/master/sql/plsql/PlSqlParser.g4#L7362) or [non_reserved_keywords_pre12c](https://github.com/antlr/grammars-v4/blob/master/sql/plsql/PlSqlParser.g4#L7748) list since it can be identifier as I guess.

Reply:
Added. Can you please take another look?